sda-commons-server-prometheus/src/main/java/org/sdase/commons/server/prometheus/PrometheusBundle.java

```diff
@@ -116,6 +119,7 @@ private static void bindJvmAndSystemMetricsToGlobalRegistry() {
     new JvmGcMetrics().bindTo(Metrics.globalRegistry);
     new ProcessorMetrics().bindTo(Metrics.globalRegistry);
     new JvmThreadMetrics().bindTo(Metrics.globalRegistry);
+    new ClassLoaderMetrics().bindTo(Metrics.globalRegistry);
```

Comment:
I wonder if we need to close all metrics as described [here](https://github.com/micrometer-metrics/micrometer/issues/1492). Should we create another subtask for that story?

Reply:
The JvmGcMetrics are the only ones that are auto closable.
I added the shutdown hook in this PR using a separate commit.

Please let me know if you prefer a separate PR

Reply:
That's fine. Thanks for the quick fix! 👍